src/Microsoft.AspNetCore.Authentication/project.json

```diff
@@ -10,6 +10,7 @@
     "keyFile": "../../tools/Key.snk"
   },
   "dependencies": {
+    "Microsoft.AspNetCore.Antiforgery": "1.0.0-*",
```

Comment:
:-1: We can have other AddXyz's. This only appears to apply to cookies, it shouldn't be pulled in for everything else.